experimental/rocm/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+################################################################################
+# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
+# nnc-iree-backend/rocm/BUILD                                                          #
```

Comment:
Is this our script that messed up this formatting? If you, maybe run bazel_to_cmake.py again?

Reply:
fixed